src/components/AudioPlayer/PlaybackControls.module.scss

```diff
@@ -5,7 +5,7 @@
   justify-content: center;
   align-items: center;
 }
-.container > button + button {
+.container > * + * {
```

Comment:
I know this is not specific. 
But this is very common util and pretty handy.

In fact we can probably turn this into a util that's integrated with our design system, with `--flow-space` css variable. 

References:
- https://every-layout.dev/layouts/stack/
- https://piccalil.li/quick-tip/flow-utility/

Reply:
I do not actually agree. We can do this to achieve the same result with higher specificity.

```
.container > .stackItem + .stackItem {
	...
}
```

You'll just have to add a className for items you want this rule to apply to.